apps/jetbrains.py

```diff
@@ -540,6 +550,25 @@ def build_text_action_keymap():
             utils.text,
             set_extend(),
         ],
+        "search symbol": idea("action GotoSymbol"),
+        "search symbol <dgndictation>": [
+            idea("action GotoSymbol"),
+            utils.text,
+            Key("enter"),
+        ],
+        "search class": idea("action GotoClass"),
+        "search class <dgndictation>": [
+            idea("action GotoClass"),
+            utils.text,
+            Key("enter"),
+        ],
+        "search file": idea("action GotoFile"),
+        "search file <dgndictation>": [
+            idea("action GotoFile"),
+            utils.text,
+            Key("enter"),
+        ],
+        "go file": idea("action GotoFile"),
```

Comment:
Dupe with "search file"

Reply:
which name do you prefer?

Reply:
I've been mostly thinking of "go" as changing the cursor location without further interaction, and "search" changing to a modal/dialog of some kind.

Reply:
will get rid of the go version